### PR TITLE
[VBLOCKS-1630] Feat: unit testing

### PR DESCRIPTION
**Pull request by kpchoy**
## Submission Checklist

 - [x] The `CHANGELOG.md` reflects any **feature**, **bug fixes**, or **known issues** made in the source code
 - [x] New unit tests and integration tests have beed added
 - [x] Code coverage is more or equal to the previous coverage
 - [x] A visual inspection of the `Files changed` tab was made prior to submitting the pull request ensuring the style guide was followed
 - [x] [CI pipeline](https://app.circleci.com/pipelines/github/twilio/twilio-voice-react-native-app) is green
 - [x] Included screenshot if necessary

## Description

Added Unit tests for store and screens

## Breakdown

- [Bulleted summary of changes]
- [eg. Updated doc]
- [eg. Added precondition to class]

## Validation

- [Bulleted summary of validation steps including local and CI pipeline]
- [eg. Added new test]
- [eg. Manually tested with both app and server]

## Additional Notes

[Any additional comments, notes, or information relevant to reviewers.]

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,4 +3,5 @@
 ## App
 * Upgrade React Native from `0.70.6` to `0.70.9`.
   * Potentially fixes iOS builds for newer versions of Xcode.
-* Added e2e tests for outgoing calls
\ No newline at end of file
+* Added e2e tests for outgoing calls
+* Added Unit Tests for store and screens
```
Comment (charliesantos) [inline CHANGELOG.md:7]:
```suggestion
* Added unit tests for store and screens
```
For consistency...

**Review by charliesantos — APPROVED** [review]
Left a minor comment. But lgtm.
Also, please add a ticket to update CI to upload artifacts after running the job. This will allow us to see coverage details on a webpage.

**Comment by kpchoy** [comment]
@charliesantos upload artifacts > https://issues.corp.twilio.com/browse/VBLOCKS-1797

**Comment by charliesantos** [comment]
> @charliesantos upload artifacts > https://issues.corp.twilio.com/browse/VBLOCKS-1797

Great!

**Pull request merged**